_includes/sections/dns.html

```diff
@@ -285,6 +285,22 @@ <h4>Terms</h4>
     <li>DNSCrypt - An older yet robust method of encrypting DNS.</li>
   </ul>
 
+  <h4>How to Validate</h4>
+
+  <ul>
+    <li>DoH / DoT
+      <ul>
+        <li>Check <a href="https://www.dnsleaktest.com/">DNSLeakTest.com</a>. <span class="badge badge-warning" data-toggle="tooltip" data-placement="bottom" data-original-title="If your DNS provider relies on anycast, the displayed ISP, hostname, or location may be misleading although valid."><i class="fas fa-exclamation-triangle"></i></span></li>
```

Comment:
It's not only the case with anycast or at least I think DNSLeaktest showed BlahDNS Swizerland as etna.switch.cz or similar.

```suggestion
        <li>Check <a href="https://www.dnsleaktest.com/">DNSLeakTest.com</a>. <span class="badge badge-warning" data-toggle="tooltip" data-placement="bottom" data-original-title="Your DNS provider may not appear with their own name, so compare the responses to what you know or can find about your DNS provider. The main point is that you shouldn't see your ISP or old unencrypted DNS provider."><i class="fas fa-exclamation-triangle"></i></span></li>
```

Reply:
The tests will always show the owner of the IP address block, which is typically the upstream ISP of the VPN provider. This suggestion is worded fine though I think. However, I would change "The main point is that you shouldn't" to "Just ensure you don't".